### DISCUSSION TITLE
Can not manage new site created

### DISCUSSION THREAD

**Discussion by qunshen** [oqtane/oqtane.framework#1553] [General]
I created a new site with a new tenanent, which has a different database rather than the master database. the site was created successfully but the control  panel does not show up! It appears that the host user doest not has the right. The host can manage the orignal site, but not the ne just created!

**Comment by sbwalker** [comment]
What alias did you use for the new site you created and what was the alias for the original site?

**Comment by qunshen** [comment]
orginal alias:http://core50.cloud123.net/, new alias:http://core50.cloud123.net/cndb4.
I checked the database, host user not created in new database

**Reply by leigh-pointer** [reply]
@qunshen  Could it be something with the DNS ?

**Reply by qunshen** [reply]
> @qunshen Could it be something with the DNS ?
it is not related to DNS, but a database issue. I checked database and find that host user not created in tenant database. I have a real-world multi-tenant with multiple database application. After I create a host user ( writing a seperate UI and stored procedure),the problem solved.

**Comment by sbwalker** [comment]
I will have to investigate further once I have time.

**Comment by sbwalker** [comment]
@qunshen which database engine are you using?

**Reply by qunshen** [reply]
sql server

**Comment by sbwalker** [comment]
I am not able to reproduce this problem. Can you please provide all of the detailed information you entered in the Add Site UI so that I can investigate further.

**Reply by qunshen** [reply]
Go site management, click "Add site", new alias name like "domian.com/xxx"---not a sperate domain name or subdomain! create a tenant ,enter a new datbase which is different from master db

**Reply by sbwalker** [reply]
So you are saying the alias for the new site you are creating must be for a domain that is totally unrelated to the current domain. That is different information than what you provided above. I will try again to reproduce the problem.

**Reply by qunshen** [reply]
the domain for the new site is the same as the current site,! I user path segment to represent different site. for example: www.cloud123.net is for current site,  www.cloud123.net/ab for new site

**Comment by qunshen** [comment]
real world example:one site:http://core50.cloud123.net/, another site:http://core50.cloud123.net/cndb4
